hlx_statics/scripts/lib-adobeio.js

```diff
@@ -179,12 +180,20 @@ export function decorateButtons(block, secondaryButtonBorderColor, secondaryButt
  * Decorates all inline codes in a container element.
  * @param {Element} element container element
  */
-export function decorateInlineCodes(element) {
+ export function decorateInlineCodes(element) {
   element.querySelectorAll('code').forEach((code) => {
     const up = code.parentElement;
+    const preElement = code.closest('pre');
     if (up.tagName !== 'PRE') {
       code.classList.add('inline-code');
     }
+    if (preElement) {
+      const wrapperDiv = document.createElement('div');
+      wrapperDiv.classList.add('code-block');
```

Comment:
I'm a bit confused about the changes done in this `decorateInlineCodes` function. I don't notice any changes comparing the inline-code from this PR's branch and one-repo:
- PR branch test url: https://fix-issues-app-builder--adp-devsite--adobedocs.aem.page/app-builder/docs/getting-started/common-troubleshooting
- one-repo branch test url: https://one-repo--adp-devsite--adobedocs.aem.page/app-builder/docs/getting-started/common-troubleshooting

Also, `code-block` is different from `inline-code`. And so I think the `code-block` logic should live in the `codeblock.js` (https://github.com/AdobeDocs/adp-devsite/blob/one-repo/hlx_statics/blocks/codeblock/codeblock.js) instead

Code-block looks like this:
<img width="913" alt="Screenshot 2024-11-20 at 7 42 15 AM" src="https://github.com/user-attachments/assets/3d75c227-f76d-483e-a7e7-6c246bb4db37">

whereas inline-code looks like 
<img width="194" alt="Screenshot 2024-11-20 at 7 42 29 AM" src="https://github.com/user-attachments/assets/8a4a9881-72a7-4be7-b769-6768d7480fa7">